src/pages/ProfilePage/views/AccountSettingsView/AccountSettingsView.tsx

```diff
@@ -10,21 +13,38 @@ import { useForm } from 'react-hook-form';
 import EditButtons from 'src/components/ProfilePage/EditButtons/EditButtons';
 import FormRow from 'src/components/ProfilePage/FormRow/FormRow';
 import TextInput from 'src/components/ProfilePage/TextInput/TextInput';
-
-import { profilePageSchema } from '/src/schemas/authSchemas';
+import { ProfilePageSchemaInterface } from 'src/schemas/Auth';
+import { profilePageSchema } from 'src/schemas/authSchemas';
 
 import styles from './AccountSettingsView.module.css';
 
-const AccountSettingsView = () => {
-  const [formData, setFormData] = useState({ nameEditable: false, passwordEditable: false, tempImage: '' });
+enum Fields {
+  Name = 'name',
+  Password = 'password',
+}
+
+interface FormData {
+  nameEditable: boolean;
+  passwordEditable: boolean;
+  tempImage: string | File;
+  image?: File;
+  namePrev?: string;
+  passwordPrev?: string;
+  email?: string;
```

Comment:
By the way, is`email` even needed in `FormData` and `ProfilePageSchemaInterface`? Could it not be skipped from the interfaces?